app-ng/src/entry-point.lisp

```diff
@@ -107,52 +135,45 @@ Copyright (c) 2016-2019 Rigetti Computing.~2%")
     (swank:create-server :port swank-port
                          :dont-close t))
 
-  (when safe-include-directory
-    (setf (gethash "safe-include-directory" **config**) safe-include-directory))
-  
+  (when (not (null simulation-method))
```

Comment:
Suggestion: `(when (not ...))` --> `(unless ...)`

Reply:
Good eye! Actually it should be "impossible" for `simulation-method` to be null here due to defaults + type checking done during config parsing. In fact, if it is null here, it's an error because the default global binding is also `nil`.

I've added some more tests to verify default handling and switched to `ASSERT`ing `SIMULATION-METHOD` is non-null here.